cmd/release-controller/sync_verify.go

```diff
@@ -30,11 +31,36 @@ func (c *Controller) ensureVerificationJobs(release *Release, releaseTag *imagev
 				}
 			}
 
+			jobRetries := 0
 			if status, ok := verifyStatus[name]; ok {
+				jobRetries = status.Retries
 				switch status.State {
-				case releaseVerificationStateFailed, releaseVerificationStateSucceeded:
-					// we've already processed this, continue
+				case releaseVerificationStateSucceeded:
 					continue
+				case releaseVerificationStateFailed:
+					jobRetries++
+					if verifyType.Optional || jobRetries > verifyType.MaxRetries {
+						continue
+					}
+					// find the next time, if ok run.
+					if status.TransitionTime != nil {
+						backoffDuration := time.Minute * 0
+						backoffCap := time.Minute * 15
+						backoffStep := time.Minute * 2
+						if jobRetries != 1 {
+							backoffDuration = backoffStep * (1 << uint(jobRetries-1))
+							if backoffDuration > backoffCap {
+								backoffDuration = backoffCap
+							}
+						}
+						backoffTime := status.TransitionTime.Add(backoffDuration)
+						currentTime := time.Now()
+						if currentTime.Before(backoffTime) {
```

Comment:
This won't work unless you queue the next retry time.  When you're before the backoff you have to add a `queue.AddAfter()` to when we will run this particular entry again (which is now + backoff)

Reply:
Otherwise you won't retry until the next time someone mutates the image stream or the resync interval.